core/ui/src/main/kotlin/nl/q42/template/ui/theme/TemplateTheme.kt

```diff
@@ -37,19 +35,24 @@ fun TemplateTheme(
     content: @Composable () -> Unit
 ) {
 
-    CompositionLocalProvider(
-        LocalTemplateTypography provides typography,
-        LocalTemplateColorScheme provides if (darkTheme) TemplateColorSchemeDark else TemplateColorSchemeLight,
-        LocalTemplateShapes provides shapes,
-        /** configures the ripple for material components */
-        LocalRippleConfiguration provides TemplateRippleConfiguration,
-        /** needed for non-material components to have a material ripple. eg [Modifier.clickable] */
-        LocalIndication provides TemplateRipple,
-        /** merges the platform style with our type, @see [ProvideTextStyle] for more context */
-        LocalTextStyle provides LocalTextStyle.current.merge(typography.body),
-        LocalContentColor provides colors.textPrimary,
-        content = content
-    )
+    MaterialTheme(
```

Comment:
Not so sure about wrapping MaterialTheme, as the philosophy before has been to specify all colors explicitly. 

With this addition, composables could get colors applied from Material implicitly. 

In this PR itr is used for the background color of the dialog. But it is of course an option to define a `surfaceElevation3` color in the color tokens and assign sensible values for it

https://m3.material.io/styles/elevation/tokens